packages/integrations/vercel/src/lib/web-analytics.ts

```diff
@@ -0,0 +1,56 @@
+import type { AnalyticsProps } from '@vercel/analytics';
+import { fileURLToPath } from 'url';
+
+export type VercelWebAnalyticsConfig = {
+	enabled: boolean;
+	config?: Omit<AnalyticsProps, 'beforeSend'>;
+};
+
+async function getWebAnalyticsFunctions(root: URL) {
+	try {
+		const files = await Promise.all([
+			import(/* @vite-ignore */ fileURLToPath(new URL('./vercel-web-analytics.ts', root))).catch(
+				() => undefined
+			),
+			import(/* @vite-ignore */ fileURLToPath(new URL('./vercel-web-analytics.js', root))).catch(
+				() => undefined
+			),
+		]);
```

Comment:
I believe it's healthier first to check if these files exist on disk e.g. [`fs.existSync`](https://nodejs.org/api/fs.html#fsexistssyncpath), and then import the ones that exist.

Also, we should log a warning if **both** files exist. 

So we don't need to rely on `Promise.all` and a `try/catch`.